modules/core/src/main/scala/lucuma/odb/api/schema/RefinedSchema.scala

```diff
@@ -4,8 +4,9 @@
 package lucuma.odb.api.schema
 
 import cats.syntax.all._
+import eu.timepit.refined.types.all.PosBigDecimal
 import eu.timepit.refined.types.all.PosInt
-import sangria.schema.{InputType, IntType, ScalarAlias}
+import sangria.schema.{InputType, IntType, BigDecimalType, ScalarAlias, ScalarType}
 import sangria.validation.ValueCoercionViolation
 
 trait RefinedSchema {
```

Comment:
Maybe `NonEmptyString` in `GeneralSchema` belongs here too.

Reply:
Indeed, I didn't want to move it but I thought the same